qcfractal/interface/collections/dataset.py

```diff
@@ -169,6 +165,69 @@ def _add_history(self, **history: Dict[str, Optional[str]]) -> None:
 
         self.data.history.add(tuple(new_history))
 
+    def list_values(self,
+                    method: Optional[str] = None,
+                    basis: Optional[str] = None,
+                    keywords: Optional[str] = None,
+                    program: Optional[str] = None,
+                    driver: Optional[str] = None,
+                    name: Optional[str] = None,
+                    native: Optional[bool] = None,
+                    force: bool = False):
+        """
+        Lists available data that may be queried with get_values.
+        Results may be narrowed by providing search keys.
+
+        Parameters
+        ----------
+        method : Optional[str], optional
+            The computational method (B3LYP)
+        basis : Optional[str], optional
+            The computational basis (6-31G)
+        keywords : Optional[str], optional
+            The keyword alias
+        program : Optional[str], optional
+            The underlying QC program
+        driver : Optional[str], optional
+            The type of calculation (e.g. energy, gradient, hessian, dipole...)
+        name : Optional[str], optional
+            The canonical name of the data column
+        native: Optional[bool], optional
```

Comment:
I wanted a name where True is the default, to emphasize that the default operating mode of QCFractal is through native compute, which is why I didn't use `contributed`. `local` and `indigenous` qualify though.

Reply:
Still not sure if `native` is right, can we raise this as an issue to be resolved before 0.10 (Monday)? A little caution here is warranted since this is likely *the* keyword that people will specify a lot for ML.